plugins/kafka/kafka.ecllib

```diff
@@ -212,9 +212,10 @@ EXPORT Consumer(VARSTRING topic, VARSTRING brokers = 'localhost', VARSTRING cons
         END;
 
         // Set the offset for each partition on each node
-        result := PROJECT
+        result := NORMALIZE
             (
-                distOffsets,
+                NOFOLD(distOffsets),
```

Comment:
The goal of this function is to locally execute kafka.SetMessageOffset().  That function updates a file local to the slave on which it is running.  Updating a distributed dataset is the technique I was using to force that local execution.  This also work correctly in Roxie and hthor (when only one node is used).

I will test today and confirm that the NOFOLD is actually needed.  It may be a leftover from earlier debugging tests when I was working with PROJECT rather than NORMALIZE.